js/client-mainmenu.js

```diff
@@ -1175,6 +1225,29 @@
 				teambuilder.selectFolder(teamFormat);
 			}
 		},
+		selectFolder: function (key) {
+			var sourceEl = this.sourceEl;
```

Comment:
why are you declaring variables for each of these as opposed to just doing `{team: this.team, ...}` etc when you call `addPopup` (ditto in `foldersToggle`)?

Reply:
I guess because youre copying the pattern from `moreTeams`? I dont know the pattern was that great to begin with.

Reply:
Yessir, simply to follow design pattern @scheibo

Reply:
I can change this all if needed

Reply:
@Zarel - can you explain this pattern of defining `var`s for seemingly no reason (here and below)

Reply:
I don't remember; I assume it was just to make their use easier. It's probably something someone else did to copy my earlier code,  which defines `team` and `format` because they're used a lot and omitting `this.` makes them more readable?

Reply:
I would definitely recommend not defining short variables if they're only used once.

Reply:
I resolved this.